tests/tools/test_dist_part.py

```diff
@@ -111,12 +113,16 @@ def test_chunk_graph(num_chunks):
                     assert isinstance(int(num2), int)
 
         # check node/edge_data
+        suffix = 'npy' if data_fmt=='numpy' else 'parquet'
```

Comment:
`parquet` is the common suffix that indicates it's PARQUET files?

Reply:
It looks a common preference, but some other suffix like ```parq``` is also used. As we use ```pyarrow``` to resolve the file , and its official doc use ```parquet```, let use it to keep consistent.